doc/flatpak-config.xml

```diff
@@ -70,7 +70,8 @@
                 <listitem><para>
                    This key is used when languages is not set, and it defines extra locale
                    extensions on top of the system configured languages. The value is a
-                   semicolon-separated list of two-letter language codes.
+                   semicolon-separated list of two-letter language codes or locale identifiers
```

Comment:
I guess since we're breaking API anyway it's a good time to consider if we want to rename this config key from `extra-languages` to `extra-locales` as @treitter suggested.

Reply:
Oh, yeah, forgot about that one

Reply:
if we call it extra-locales, would it be surprising that we also accept languages?

I think changing the name of the key is much more of an api break than silently accepting more values

Reply:
You're right, probably best to stick with extra-languages